### DISCUSSION TITLE
Time limits ignored for recurring transactions

### DISCUSSION THREAD

**Discussion by Syncena** [firefly-iii/firefly-iii#9293] [General]
Before I file a bug report, thought I throw this out here first in case it's a PEBKAC. Apologies if my description is a little vague - my wife uses FireFly-iii daily; I'm only the installer/administrator.

In the following example, my wife sets up a recurring transaction set to repeat three times:

![firefly-rectr1](https://github.com/user-attachments/assets/70bd2df9-c857-43f6-8011-1272ee94d706)

However, when viewing the recurring transaction, the repetition count is not shown, and the transaction will continue to be repeated beyond the required limit:

![firefly-rectr2](https://github.com/user-attachments/assets/d0bd0ffa-dfbc-4c9e-9f95-eb27991fc5a9)

Note: this was noticed because she had a genuine recurring monthly transaction with repeat count set to 5 which was applied a sixth time today.

This is on v6.1.21.

Steve.

**Comment by JC5** [comment]
Nice, that could be a bug. Thanks for opening an issue. I'll check it out!

**Reply by Syncena** [reply]
Thanks! Bug #9294
